trulens_eval/trulens_eval/app.py

```diff
@@ -731,7 +749,7 @@ def main_input(
             callable_name(func), all_args
         )
 
-        return None
+        return ""
```

Comment:
Perhaps this can fallback to just returning str(all_args) maybe with a prefix "Could not determine main input/output of {str(all_args)}." ?